libraries/Microsoft.Bot.Builder.LanguageGeneration/ErrorListener.cs

```diff
@@ -9,7 +10,19 @@ public class ErrorListener : BaseErrorListener
     {
         public override void SyntaxError([NotNull] Antlr4.Runtime.IRecognizer recognizer, [Nullable] IToken offendingSymbol, int line, int charPositionInLine, [NotNull] string msg, [Nullable] RecognitionException e)
         {
-            throw new Exception($"[ERROR]: syntax error at line {line}:{charPositionInLine} {msg}");
+            var lineStrArray = offendingSymbol.InputStream.GetText(new Interval(0, int.MaxValue)).Split('\n');
```

Comment:
you should still output this line number and char number, i expect this error message will be

[Error] syntax error at line 4, char 5-10
line3:   # templateName
line4:    - [t2]
               ^^ no such template
line5：# another template